build.gradle

```diff
@@ -0,0 +1,51 @@
+buildscript {
+    repositories {
+        mavenCentral()
+    }
+}
+
+plugins {
+    id 'org.jetbrains.intellij' version '0.2.17'
+    id 'org.jetbrains.kotlin.jvm' version '1.1.50'
+}
+
+ext {
+    kotlinVersion = '1.1.50'
+    jBehaveVersion = '4.0.5'
+    hamcrestVersion = '1.3'
+    guavaVersion = '18.0'
+    jUnitVersion = '4.11'
+    easyTestingVersion = '1.2.5'
+    easyTestingAssertCoreVersion = '2.0M10'
+    mockitoVersion = '1.9.5'
+}
+
+repositories {
+    mavenCentral()
+}
+
+apply plugin: 'java'
+apply plugin: 'kotlin'
+apply plugin: 'org.jetbrains.intellij'
+
```

Comment:
`apply plugin: 'idea'`
can be useful too.
In order to set this up locally this should suffice (maybe add it to README.md)
```
git clone ...
./gradlew idea
```
Open IntelliJ and Open -> select folder -> done!